flask_resty/spec/utils.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+__all__ = ['ref',
```

Comment:
What's the point of this `__all__`? I think it'd be better to just not use a wildcard import in `__init__`.